### PR TITLE
ci: add a checklist in the pull request template

### PR DESCRIPTION
**Pull request by elopio**
LP: #1649892

### CODE REVIEW

**Comment by elopio** [comment]
Here's a simple change for your consideration.

**Comment by sergiusens** [comment]
Is there github documentation for this?

**Comment by elopio** [comment]
@sergiusens https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Thanks for helping us make a better Snapcraft!
+
+Checklist:
+
+- Have you signed the contributor licence agreement?
+  https://www.ubuntu.com/legal/contributors
+
+- Have you reported a bug for the problem you are fixing?
+  https://bugs.launchpad.net/snapcraft/+filebug
```
Comment (elopio) [inline PULL_REQUEST_TEMPLATE.md]: @kyrofa I'm not quite sure how to do that in a nice way. Please check my push. I'm open to suggestions.

Comment (kyrofa) [inline PULL_REQUEST_TEMPLATE.md]: Yeah I think that works just fine :smiley: .

Comment (sergiusens) [inline PULL_REQUEST_TEMPLATE.md]: I would add something about searching for existing bugs this would fix and if not log a new bug.

Comment (elopio) [inline PULL_REQUEST_TEMPLATE.md]: @sergiusens what about now?

**Pull request merged**